backend/danswer/danswerbot/slack/handlers/handle_message.py

```diff
@@ -47,6 +52,25 @@ def send_msg_ack_to_user(details: SlackMessageInfo, client: WebClient) -> None:
     )
 
 
+def add_user_if_not_exists(email: str, db_session: Session) -> User:
```

Comment:
We like to throw these functions that interface with the DB models (and that may be reuseable) in the related file under backend/db. Makes organization/finding things easy and helps prevents circular imports.